app.py

```diff
@@ -1,11 +1,43 @@
 import flask
+from flask import send_file
 
```

Comment:
It's better to use render_template for displaying page contents. Here's why:

- Your templates can be extended via Jinja, meaning you only have to write boilerplate code once.
- You can serve dynamic content, whereas send_file simply returns static content.
- You do not need to hard code absolute path since Flask looks for templates under the templates directory.

Reply:
Hey Jay,
I made the changes as you suggested. I think this is a good idea.